website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -198,19 +212,24 @@ database and glacier on a file version when doing a recovery.
     "collections": [
       {
         "collectionId": "MOD14A1__061",
+        "shortname": "AST_L1A",
+        "collection_version": "AST_L1A_1.0",
         "granules": [
           {
             "granuleId": "MOD14A1.061.A23V45.2020235",
-            "date": "TBD",
-            "createDate": "2020-01-01T23:00:00+00:00",
+            "cumulus_create_time": "2020-01-01T23:00:00+00:00",
```

Comment:
Currently Input uses camelCase and output uses both camelCase and underscores. Since Cumulus seems to like camel case, it may be simplest to leave output that way. Thoughts?

Reply:
changed to camelCase format per our discussion